cmake/Common.cmake

```diff
@@ -21,24 +23,52 @@ function(package_set_postinst_prerm LIB_NAMES LIB_DIRS INCLUDE_DIRS)
         set(POSTINST_SOURCE "${POSTINST_SOURCE}\nln -sr ${lib_dir}/lib${lib_name}.so ${lib_dir}/../../lib/lib${lib_name}.so")
         set(POSTINST_SOURCE "${POSTINST_SOURCE}\nln -sr ${lib_dir}/cmake/${lib_name} ${lib_dir}/../../lib/cmake/${lib_name}\n")
 
+        set(PREINST_SOURCE "${PREINST_SOURCE}\nif [ $1 == 2 ]; then")
+        set(PREINST_SOURCE "${PREINST_SOURCE}\n\trm /etc/ld.so.conf.d/${lib_name}.conf")
```

Comment:
Would it be better to use "rm -f" instead of "rm", so it doesn't fail if the file does not exist?

Reply:
Agreed.